OpenRA.Mods.Common/Traits/World/SpawnMapActors.cs

```diff
@@ -35,10 +35,13 @@ public void WorldLoaded(World world, WorldRenderer wr)
 			{
 				var actorReference = new ActorReference(kv.Value.Value, kv.Value.ToDictionary());
 
-				// If there is no real player associated, don't spawn it.
-				var ownerName = actorReference.Get<OwnerInit>().InternalName;
-				if (!world.Players.Any(p => p.InternalName == ownerName))
-					continue;
+				// If an actor's doesn't have a valid owner transfer ownership to neutral
+				var ownerInit = actorReference.Get<OwnerInit>();
+				if (!world.Players.Any(p => p.InternalName == ownerInit.InstanceName))
```

Comment:
Shouldn't you check whether ownerInit is null first before trying to check its member?
`if (ownerInit != null && !world.Players.Any(p => p.InternalName == ownerInit.InstanceName))`

Reply:
no, all actors must have an owner field

Reply:
though I guess this new behaviour would allow not having it

Reply:
`.Get` will throw if it doesn't exist.